README.md

```diff
@@ -0,0 +1,203 @@
+# Wikidata Reference Island
+
+## Glossary
+
+The following terms will be used throughout this document, their meanings are as follows:
+
+**Pipe <#>:** Represents a sequential step in the main data pipeline, which processes Wikidata item serializations into a formatted list of potential references. 
+
+**SS<#>:** Represents a "side service", to provide additional data to steps in the main pipeline, which will aid in making decisions or filter and format potential references.
+
+## General Agreements
+
+* The output formats for each step in the pipeline is to be formatted and encoded in `jsonl` with the exception of [Pipe 5](pipe-5).
+
+## Pipe 1: Find unreferenced statements and URLs for a given list of Wikidata items<a name="pipe-1"></a>
+
+This step will take in the following inputs, and will output the data expected in [Pipe 2](#pipe-2).
+
+* `<varname>`: List of strings representing white listed Wikidata external id properties, same as output of [SS1](#ss-1).
```

Comment:
"white listed" is being used inconsistently. I'd go with "whitelisted" which is totally fine. (Fun fact: If you read Dracula, it uses "to-morrow" instead of "tomorrow". It's natural for words to lose the hyphen).

Reply:
That is interesting, yeah, my spellcheck was bugging my with the nin hyphenated version, but I'll just force it to learn this spelling.

Reply:
This appears to now be fixed